docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: '2'
+services:
+  ui-tests:
+    build: .
+    environment:
+      REDASH_SERVER_URL: http://redash:5000
+    links:
+      - "server:redash"
+    entrypoint: ./docker-entrypoint.sh
```

Comment:
Hmm I'm not sure I understand why we overwrite ``ENTRYPOINT`` and need this shell script? 🤔

Reply:
This allows us to just ```docker-compose run``` pytest within the container. We can just use ```docker-compose run --rm ui-tests ui_test```.

Reply:
Thinking about this more, would it be better to just tell the user to do a ```docker-compose run ui-tests``` and by default that will run pytest command. Only consideration is that it wouldn't run anything else, including flake8.

Reply:
We need to be careful about mixing up workflows here:

- running the tests is the default operation and what users of this project will do (via CI or manually)
- developing tests and checking our code is what contributors will do

I don't think we need a way to run flake8 inside of the Docker container, because we will have the project checked out and set up on our local machine.

``docker-compose run ui-tests`` would run the the tests 👍

Reply:
Agreed, so no need for the entrypoint then